### PR TITLE
[Python] Updates to implementing-a-concept-exercise.md

### PR DESCRIPTION
**Pull request by BethanyG**
Updated/edited `languages/python/reference/implementing-a-concept-exercise.md`.

### CODE REVIEW

File: languages/python/reference/implementing-a-concept-exercise.md [hunk 1]
```diff
@@ -38,42 +40,106 @@ languages
...
 
-How to create the files common to all tracks is described in the [how to implement a concept exercise document][how-to-implement-a-concept-exercise].
+## Step 2: Add exercise documentation files
+
+How to create the files common to all tracks is described in the [how to implement a concept exercise document][how-to-implement-a-concept-exercise]. All of these files are written in [Markdown][commonmark], and you can find the [Exercism Formatting and Style Guide][style-guide] here. Please pay close attention to the [auto-formatting][auto-formatting] section, to avoid CI test failures.
+
+As a reminder, code elements (functions, keywords, operators) should be wrapped in backticks:
+
+```python
+A `set` is an unordered collection of distinct hashable objects.  To add something to a `set`, use `set.add()`
```
Comment (valentin-p) [inline languages/python/reference/implementing-a-concept-exercise.md:70]:
```suggestion
A `set` is an unordered collection of distinct hashable objects. To add something to a `set`, use `set.add()`
```

File: languages/python/reference/implementing-a-concept-exercise.md [hunk 2]
```diff
@@ -38,42 +40,106 @@ languages
...
+i
+m
+b
+u
+k
+t
+u
+```
+
+For resource links, we strongly favor linking into relevant parts of the [Python docs][python-docs] as a first source, with other useful and interesting links as a follow-on. Please avoid any pawalled or subscription-based/signup required links.
```
Comment (valentin-p) [inline languages/python/reference/implementing-a-concept-exercise.md:101]:
```suggestion
For resource links, we strongly favor linking into relevant parts of the [Python docs][python-docs] as a first source, with other useful and interesting links as a follow-on. Please avoid any paywalled, subscription-based or signup-required links.
```

**Review by valentin-p — APPROVED** [review]
LGTM. Could you just check/commit my two comments?

File: languages/python/reference/implementing-a-concept-exercise.md (hunk 2 above)
Comment (BethanyG) [reply languages/python/reference/implementing-a-concept-exercise.md:101]: Many thanks for those!

File: languages/python/reference/implementing-a-concept-exercise.md [hunk 3]
```diff
@@ -38,42 +40,106 @@ languages
...
 [reference]: ../../../reference
 [how-to-implement-a-concept-exercise]: ../../../docs/maintainers/generic-how-to-implement-a-concept-exercise.md
 [docs-concept-exercises]: ../../../docs/concept-exercises.md
 [docs-rationale-for-v3]: ../../../docs/rationale-for-v3.md
 [docs-features-of-v3]: ../../../docs/features-of-v3.md
 [anatomy-of-a-concept-exercise]: https://www.youtube.com/watch?v=gkbBqd7hPrA
+[pep8]: https://www.python.org/dev/peps/pep-0008/
+[example-todo]: ../exercises/concept/guidos-gorgeous-lasagna/lasagna.py
+[example-stub]: ../exercises/concept/ghost-gobble-arcade-game/arcade_game.py
+[example-testfile]: ../exercises/concept/little-sisters-essay/str_methods_test.py
```
Comment (BethanyG) [inline languages/python/reference/implementing-a-concept-exercise.md:136]:
```suggestion
[example-testfile]: ../exercises/concept/little-sisters-essay/string_methods_test.py
```

**Pull request merged**